src/main/resources/log4j2.xml

```diff
@@ -16,9 +16,6 @@
             <SizeBasedTriggeringPolicy size="10 MB" />
             <DefaultRolloverStrategy max="10"/>
         </RollingFile>
-        <Console name="console" target="SYSTEM_OUT">
```

Comment:
Please re-add the Console appender, it is used by the infrastructure to pull all the logs into a central location.